src/main/java/org/mitre/synthea/export/rif/OutpatientExporter.java

```diff
@@ -206,18 +159,18 @@ long export(Person person, long startTime, long stopTime) throws IOException {
           exporter.rifWriters.writeValues(BB2RIFStructure.OUTPATIENT.class, fieldValues);
         }
 
-        if (claimLine == 1) {
-          // If claimLine still equals 1, then no line items were successfully added.
-          // Add a single top-level entry.
-          fieldValues.put(BB2RIFStructure.OUTPATIENT.CLM_LINE_NUM, Integer.toString(claimLine));
-          fieldValues.put(BB2RIFStructure.OUTPATIENT.REV_CNTR_DT,
-                  RIFExporter.bb2DateFromTimestamp(encounter.start));
-          // 99241: "Office consultation for a new or established patient"
-          fieldValues.put(BB2RIFStructure.OUTPATIENT.HCPCS_CD, "99241");
-          setClaimCosts(fieldValues, encounter.claim.totals);
-          setLineItemCosts(fieldValues, encounter.claim.totals);
-          exporter.rifWriters.writeValues(BB2RIFStructure.OUTPATIENT.class, fieldValues);
-        }
+        // Add a total charge entry.
+        fieldValues.remove(BB2RIFStructure.OUTPATIENT.REV_CNTR_IDE_NDC_UPC_NUM);
+        fieldValues.remove(BB2RIFStructure.OUTPATIENT.REV_CNTR_NDC_QTY);
+        fieldValues.remove(BB2RIFStructure.OUTPATIENT.REV_CNTR_NDC_QTY_QLFR_CD);
+        fieldValues.put(BB2RIFStructure.OUTPATIENT.CLM_LINE_NUM, Integer.toString(claimLine));
+        fieldValues.put(BB2RIFStructure.OUTPATIENT.REV_CNTR_DT,
+                RIFExporter.bb2DateFromTimestamp(encounter.start));
+        fieldValues.put(BB2RIFStructure.OUTPATIENT.REV_CNTR, "0001");
+        // 99241: "Office consultation for a new or established patient"
+        fieldValues.put(BB2RIFStructure.OUTPATIENT.HCPCS_CD, "99241");
```

Comment:
Were all total charges associated with `99241`?

Reply:
Good catch. In the analysis most (1,582,011) total charge line items had an empty HCPCS code. The remaining 722 were sprinkled over 47 other codes of which 210 were for HCPCS 99241. I think we could safely use an empty code here instead since that accounts for 99.95% of the total charge line items.